packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx

```diff
@@ -149,6 +157,25 @@ export const WalletManagementTile = () => {
   const { isEnabled: isStripeEnabled } = useFlag(
     FeatureFlags.BUY_AUDIO_STRIPE_ENABLED
   )
+  const coinbaseAllowedCountries = (
```

Comment:
lot of pretty low level logic here, can we pull these into a `isRegionSupported` hook even if we're not reusing outside of this component?